client/components/AttributionEditor/AttributionDetailControls.tsx

```diff
@@ -98,7 +98,7 @@ const AttributionDetailControls = (props: Props) => {
 			<div className="right-details">
 				<InputGroup
 					placeholder="Affiliation"
-					defaultValue={affiliation}
+					defaultValue={affiliation ?? undefined}
```

Comment:
I do this in a number of places where I cannot change the definition from e.g. `string | undefined` to `string | null | undefined`. Kind of ugly, and not very nice to have to continue doing. If you have a better idea let me know!